docs/setup/checkpoints.md

```diff
@@ -28,12 +28,21 @@ under the License.
 
 ## Overview
 
-TBD
-
+Checkpoints make state in Flink fault tolerant by allowing to recover state
+and positions in the streams to give the application the same semantics as
+a failure-free execution.
```

Comment:
Checkpoints make state in Flink fault tolerant by allowing state
and the corresponding stream positions to be recovered, 
thereby giving the application the same semantics as a failure-free execution.